handlers/load_test_preferences_handler.go

```diff
@@ -75,3 +80,197 @@ func (h *Handler) LoadTestPrefencesHandler(w http.ResponseWriter, req *http.Requ
 
 	_, _ = w.Write([]byte("{}"))
 }
+
+type custTestConf struct {
+	Val *SMPS.PerformanceTestConfig
+}
+
+func (c *custTestConf) MarshalJSON() ([]byte, error) {
+	m := jsonpb.Marshaler{
+		EmitDefaults: true,
+	}
+	val, err := m.MarshalToString(c.Val)
+	return []byte(val), err
+}
+
+// UserTestPreferenceHandler is used for persisting load test preferences
+func (h *Handler) UserTestPreferenceHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+
+	if req.Method == http.MethodPost {
```

Comment:
Can we create two different handlers for Post & Delete rather than putting complete boilerplate code in the conditionals?

Reply:
Totally, I was considering this. I was actually thinking we must use gorilla/mux.  That will allow us to build better REST APIs, check [this](https://gist.github.com/ssaumyaranjan7/5732157a21835c9f08a59be6a9317c0b#file-bookstore-routes-go-L10) out. What's your opinion on this?

Reply:
Use of gorilla mux is a great improvement. 👍